source/scripts/script.js

```diff
@@ -14,20 +9,20 @@ const images = ["./img/foodPics/indian.jpeg", "./img/foodPics/vegan.jpeg", "./im
     "./img/foodPics/gluten-free.jpeg", "./img/foodPics/italian.jpeg", "./img/foodPics/japanese.jpeg", "./img/foodPics/american.jpeg", "./img/foodPics/vegetarian.jpeg",
     "./img/foodPics/thai.jpeg", "./img/foodPics/chinese.jpeg", "./img/foodPics/korean.jpeg", "./img/foodPics/vietnamese.jpeg", "./img/foodPics/african.jpeg", "./img/foodPics/middleEastern.jpeg"];
 
-
-//on enter for search, call search function
-document.addEventListener('keydown', function (event) {
-    if (event.key === 'Enter') {
-        search();
-    }
-});
-
 window.addEventListener('DOMContentLoaded', init);
 
 async function init() {
     showHome();
     createCategoryCards();
     showRecipePage();
+    const clearBtn = document.getElementById('clear-btn');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8408391)
```suggestion
    const clearBtn = document.getElementById("clear-btn");
```